src/main/java/org/broadinstitute/hellbender/tools/spark/sv/RunSGAViaProcessBuilderOnSpark.java

```diff
@@ -62,14 +62,19 @@
               optional  = true)
     public boolean runCorrectionSteps = false;
 
+    @Argument(doc       = "To enable stdout & stderr capture of the SGA processes or not.",
```

Comment:
I'd change the wording of the doc and arg names a little bit. Maybe something like:

doc = "If true, capture the stdout and stderr of the SGA processes along with the assembly output (valuable for debugging but hurts performance)"
shortName=captureStdIO
fullName=captureSGAStdOutAndError